added a note for the issue with libgdbm3 (#12282)
meshery/meshery: docs/pages/project/contributing/contributing-docs.md

  sudo apt-get install autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm3 libgdbm-dev
  </code>
  </div></pre>

> {% include alert.html type="info" title="Note" content="Latest versions of debian based OS does not support libgdbm3, so if you encounter any issue with its installation then try replacing it with libgdbm6 or libgdbm5." %}

Reviewer: ```suggestion
{% include alert.html type="info" title="Note" content="The latest versions of Debian-based OS'es do not support <code>libgdbm3</code>, so if you encounter an issue with it's installation, try replacing <code>libgdbm3</code> with <code>libgdbm5</code> or <code>libgdbm6</code>." %}
```
